sencha-workspace/SlateTasksStudent/app/view/TaskTree.js

```diff
@@ -171,39 +195,30 @@ Ext.define('SlateTasksStudent.view.TaskTree', {
         }
     },
 
+
     // config handlers
     updateCourseSection: function(val) {
         var me = this;
 
         me.fireEvent('coursesectionchange', me, val);
     },
 
-    applyData: function(data) {
-        data.statusClasses = this.getStatusClasses();
-        return data;
-    },
-
-    updateData: function(data) {
-        this.update(data);
-    },
 
     // event handlers
     onTreeClick: function(ev, t) {
         var me = this,
             target = Ext.get(t),
-            parentEl, recordId;
+            parentEl = target.up('.slate-tasktree-item'),
+            recordId;
 
         if (target.is('.slate-tasktree-nub.is-clickable')) {
```

Comment:
@BillClinton use `if (ev.getTarget('.slate-tasktree-nub.is-clickable'))` as this test will be immune to being broken by sub-elements getting added to the HTML that could end up being the target of the click event